src/app/shared/group-title/group-title.component.pug

```diff
@@ -0,0 +1,11 @@
+.row.group-border
+    .col-9.pb-24.pt-24.z-1000
+        .row.m-sm-0.pb-3(#titlerow="")
+            h1.title {{ ig.name }}
+        .row.m-sm-0.item-last-updated
+            span {{ ig.total }} items
+            span &nbsp;&nbsp;•&nbsp;&nbsp;
```

Comment:
I'd like @codiak 's approval here on using no break space, instead of CSS. This works for me though 👍

Reply:
Oooh, a class for the dot would be much better. Something inline-block with a margin 🙏